crates/subspace-proof-of-time/src/aes.rs

```diff
@@ -48,7 +48,7 @@ fn create_generic(seed: &PotSeed, key: &PotKey, checkpoint_iterations: u32) -> P
 pub(crate) fn verify_sequential(
     seed: &PotSeed,
     key: &PotKey,
-    checkpoints: &[PotCheckpoint],
+    checkpoints: &[PotProof],
```

Comment:
Checkpoints are basically the same as proofs, just run for a fraction of total iterations and use the last checkpoint as a proof that actually matters. I didn't want to have both checkpoints and proofs at the same time since they are too similar and using checkpoints elsewhere (like `proof_of_time: PotCheckpoint`) was more awkward than the other way around.